### PR TITLE
Init implementation

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**
- added Spark Operator chart with configs
- added service account config, which will be used by users to manage spark apps, pods, svcs in operator's deployment namespace
- added helm & kube CLI clients (lots of copied from `platform-operator` :slightly_smiling_face: )
- added CLI to deploy list deployments, deploy new instance, generate kubectl config for SA (which will be given to the user)

How we will use it:
1. request comes from the user to get Spark Cluster.
2. we use CLI of spark-operator to deploy one in some namespace (we select one)
3. we generate kubectl config for the user and give it for him
4. the user could manage applications from the local machine or from platform job

todos:
- add tests
- validate on onprem (prior I was using minikube exclusively) 
- validate on more realistic use-case from @andriihomiak
- in other PRs -- more improvements from https://docs.google.com/document/d/16TImGhfUOJAmBh5rUxy-bKCokQ23Cs9kQtGDp1PKcuI/edit

### CODE REVIEW

**Comment by YevheniiSemendiak** [comment]
IDK why I cannot tag @andriihomiak in reviewers, but will leave a msg here.

Please, take a look, guys.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+include k8s.mk
```
Comment (xomiachuna) [inline Makefile:1]: This file seems to be missing

Comment (YevheniiSemendiak) [reply Makefile:1]: correct, fixed in [b0c790d](https://github.com/neuro-inc/mlops-spark-operator/pull/1/commits/b0c790d21249f3c4a713ed7c02a0ffd37dd3fc6e)

**Comment by xomiachuna** [comment]
I'm sadly unable to get kubeconfig generation to work:
```
$ platform-spark get-kubectl-config -n spark-test -o kubeconfig
Expecting value: line 1 column 1 (char 0)
```
This also happens upon `install` (installation seems to be successful tho)

Increasing the verbosity does not provide any more meaningful info on the error (no stacktrace)

**Pull request merged**